.gitignore

```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /package-lock.json
+/pnpm-lock.yaml
```

Comment:
What is this @VirAlex, is it needed?

Reply:
Yes because I'm using pnpm to replace npm wich is faster and better ;) 
https://www.atatus.com/blog/npm-vs-yarn-vs-pnpm/
Just for me ;)